.github/workflows/run_tests.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     name: Unit test [py${{ matrix.python }} ${{ matrix.os }}]
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
Now that the PR is in working order, is this still needed?

Reply:
I think it is useful to keep. CI resources are plentiful. With the default `fail-fast=true`, a developer can solve only 1 error per commit-run iteration. With `fail-fast=false`, you can see all the different errors that occur in different matrix elements and make faster progress on solving them.